app/views/administrate/application/index.html.erb

```diff
@@ -55,6 +55,15 @@ It renders the `_table` partial to display details about the resources.
   </div>
 </header>
 
+<% page.scope_groups.each do |group| %>
+<div class="scope-buttons">
```

Comment:
I've tried to do what you told me with this commit:

https://github.com/nando/administrate/commit/8649b8773e132

As I told you before I'm a completely Sass newbie and also VERY bad with design and CSS, so please fix what i've done as you like. :)

And thanks!!!